sources/Interop/Windows/Windows/um/coml2api/STGOPTIONS.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/coml2api.h in the Windows SDK for Windows 10.0.20348.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop.Windows;
+
+/// <include file='STGOPTIONS.xml' path='doc/member[@name="STGOPTIONS"]/*' />
```

Comment:
Working on stabilizing my docs generation tool so it can be part of the process here. It's not quite ready yet.